.github/workflows/deploy.yml

```diff
@@ -5,13 +5,34 @@ on:
     tags:
       - '*'
 
+env:
+  IMAGE_NAME: quay.io/mittwald/kube-mail
+  REGISTRY_URL: quay.io/mittwald
+
 jobs:
   deploytagged:
     name: Deploy Helm chart
     runs-on: ubuntu-latest
     if: github.repository == 'mittwald/kube-mail'
     steps:
+      - name: Docker registry login
+        run: docker login -u "${{ secrets.dockerLoginUsername }}" -p "${{ secrets.dockerLoginPassword }}" "${REGISTRY_URL}"
```

Comment:
Also, should `REGISTRY_URL` just be `quay.io`? Or does `docker login [...] quay.io/mittwald` also work?